test/bundlers.test.js

```diff
@@ -70,6 +70,28 @@ test('from kebabo-case to camelCase', (t) => {
   t.end()
 })
 
+test('from @-prefixed named imports', (t) => {
+  const plugin = fp((fastify, opts, next) => {
+    next()
+  }, {
+    name: '@hello/world'
+  })
+
+  t.equal(plugin.helloWorld, plugin)
+  t.end()
+})
+
+test('from @-prefixed named kebabo-case to camelCase', (t) => {
```

Comment:
```suggestion
test('from @-prefixed named kebab-case to camelCase', (t) => {
```

Reply:
yeah, well. I oriented myself to the test cases above...